I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 642, in script call
    "Nous sommes en 1870, à Tokyo, capitale nouvellement renommée de l’Empire du Japon. Depuis peu, le pays s’ouvre aux techniques étrangères, et de nombreux occidentaux parcourent ses rues."
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
  File "game/scripts/cuisine/cuisine_ch1.rpy", line 365, in script
    clateau "À l’époque, mon père voulait que j’aie un travail prometteur, alors je suis rentrée à l’académie militaire. J’étais jeune, je pensais que ça ne durerai qu’un temps, que j’allais voyager… Mais une fois les campagnes de colonisation débutée, on peut difficilement s’extraire du corps colonial…"
  File "renpy/common/00keymap.rpy", line 411, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 411, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 430, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
RuntimeError: maximum recursion depth exceeded while pickling an object

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\renpy\renpy\bootstrap.py", line 313, in bootstrap
    renpy.main.main()
  File "C:\renpy\renpy\main.py", line 513, in main
    run(restart)
  File "C:\renpy\renpy\main.py", line 139, in run
    renpy.execution.run_context(True)
  File "C:\renpy\renpy\execution.py", line 876, in run_context
    context.run()
  File "game/script.rpy", line 642, in script call
    "Nous sommes en 1870, à Tokyo, capitale nouvellement renommée de l’Empire du Japon. Depuis peu, le pays s’ouvre aux techniques étrangères, et de nombreux occidentaux parcourent ses rues."
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
  File "game/scripts/cuisine/cuisine_ch1.rpy", line 365, in script
    clateau "À l’époque, mon père voulait que j’aie un travail prometteur, alors je suis rentrée à l’académie militaire. J’étais jeune, je pensais que ça ne durerai qu’un temps, que j’allais voyager… Mais une fois les campagnes de colonisation débutée, on peut difficilement s’extraire du corps colonial…"
  File "C:\renpy\renpy\ast.py", line 678, in execute
    renpy.exports.say(who, what, interact=self.interact, *args, **kwargs)
  File "C:\renpy\renpy\exports.py", line 1203, in say
    who(what, *args, **kwargs)
  File "C:\renpy\renpy\character.py", line 1027, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "C:\renpy\renpy\character.py", line 819, in do_display
    **display_args)
  File "C:\renpy\renpy\character.py", line 568, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\renpy\renpy\ui.py", line 289, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\renpy\renpy\display\core.py", line 2662, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\renpy\renpy\display\core.py", line 3122, in interact_core
    renpy.exports.reload_script()
  File "C:\renpy\renpy\exports.py", line 1621, in reload_script
    renpy.game.call_in_new_context("_save_reload_game")
  File "C:\renpy\renpy\game.py", line 333, in call_in_new_context
    return renpy.execution.run_context(False)
  File "C:\renpy\renpy\execution.py", line 876, in run_context
    context.run()
  File "renpy/common/00keymap.rpy", line 411, in script
    python hide:
  File "C:\renpy\renpy\ast.py", line 882, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\renpy\renpy\python.py", line 1913, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/00keymap.rpy", line 411, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 430, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
  File "C:\renpy\renpy\loadsave.py", line 404, in save
    dump((roots, renpy.game.log), logf)
  File "C:\renpy\renpy\loadsave.py", line 48, in dump
    cPickle.dump(o, f, cPickle.HIGHEST_PROTOCOL)
RuntimeError: maximum recursion depth exceeded while pickling an object

Windows-8-6.2.9200
Ren'Py 7.1.0.882
Meiji 1.0
Mon Oct 29 15:19:33 2018
